### PR TITLE
"Cannot read property 'forEach'" error after adding Graphic Interactions via "Insert block below"

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-582

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-item-runner-qti",
-    "version": "0.22.0",
+    "version": "0.22.1",
```
Comment (clagosarias) [inline package.json:3]: I thought the version was generated when publishing the package

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

File: package.json (hunk 1 above)
Comment (habemuscode) [reply package.json:3]: I will remove it when update from develop, thanks

**Review by gabrielfs7 — APPROVED** [review]

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**